content/docs/0.15.x/api/git_provisioning/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Automatic Git Provisioning
+description: Automatically provision a Git repository when a project is created.
+weight: 20
+keywords: [git provisioning]
+aliases:
+  - /docs/0.15.x/api/git_provisioning
+---
+
+
+### Description
+
+Keptn provides an extension point that can be used to automatically provision a Git repository.
+When a project is created, Keptn will perform a call to the service that adhere to a specific HTTP REST interface. The swagger documentation for this API can be found [here](../../../../api/).
+
+Keptn will call the service that implements such interface passing the project name and the current namespace in which Keptn is ran. The service is expected to answer with the URL, username, and token to access the repository. Keptn will use this information to configure the upstream for the project.
+Similarly, when a project is deleted, Keptn will call the service implementing such an API. This can be used to deprovision the repository and perform clean up actions.
```

Comment:
```suggestion
Similarly, when a project is deleted, Keptn calls the service implementing such an API. This can be used to deprovision the repository and perform clean up actions.
```